colors/gruvbox-material.vim

```diff
@@ -1326,6 +1326,30 @@ call gruvbox_material#highlight('LightspeedPendingOpArea', s:palette.bg0, s:pale
 highlight! link LightspeedMaskedChar Purple
 highlight! link LightspeedGreyWash Grey
 " }}}
+" theHamsta/nvim-semantic-tokens {{{
```

Comment:
This is now a core Neovim feature, so IMO it should be moved close to the Treesitter highlight groups, and the reference to the (now deprecated) theHamsta/nvim-semantic-tokens repo should be removed.

Reply:
Sure, will do. I wasn't sure whether the highlight groups were the same in theHamsta's plugin and in the core implementation. I have the plugin with Neovim 0.8.1 right now.